packages/neon-core/src/rpc/RPCClient.ts

```diff
@@ -238,15 +238,25 @@ export class RPCClient {
    */
   public async getVersion(): Promise<string> {
     try {
-      const response = await this.execute(Query.getVersion());
-      const newVersion = response.result.useragent.match(versionRegex)[1];
-      this.version = newVersion;
+      const response = yield this.execute(Query_1.default.getVersion());
+      if (response && response.result && response.result.useragent) {
+        var version = response.result.useragent.split(':');
+        version[0] = version[0].substring(1);
+        version[1] = version[1].substring(0, version[1].length-1);
+        const newVersion = version[1];

```

Comment:
Instead of stripping it seperately, lets start by stripping the leading and trailing `/`. Then we can use array destructuring to clarify our intent instead of using array indices:

```
const responseLength = response.result.useragent.length;
const strippedResponse = response.result.useragent.substring(1, responseLength-1);
const [header, newVersion, ..._] = strippedResponse.split(':');
this.version = newVersion;
```